docs/templates/how-tos.md

```diff
@@ -0,0 +1,42 @@
+---
+---
+
+The how-tos should:
+
+* Either have a single task using a sequence of steps 
+* Or have multiple related short tasks (each < 2 screens long and can be done in random order)
+* May include screenshots 
+* Include only necessary conceptual information
+
+Add the title in _data/reference.yml. The title should start with verb+ing
+
+Add very short description here, which should include:
+
+* Why should I read this / what this page describes? (1 sentence)
+* Any background information I need to know (to understand what I’m going to do in this guide). (1 sentence)
+* Links to more in-depth information  
+
+Add TOC (we can omit TOC if there's only a single task):
+
+* TOC
+{:toc}
+
+
+## Before you begin
+Prerequisites (we can assume that users have completed quickstart)
+
+## Doing Task 1
+If there are multiple ways to complete a task, use a tab widget to describe it.
+
+## Doing Task 2
+
+## Doing Task N
```

Comment:
If we're planning to favor smaller, more focused docs, I'm not sure we want to encourage docs with N tasks.